CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-### 2.3.2 (Next)
+### 3.0.0 (Next)
```

Comment:
This doesn't look like a breaking change to me. It's mostly a backward compatible feature. Any reason why you bumped the major rather than the minor that I'm missing? :thinking:

Reply:
My rationale was that this would require a user to upgrade/modify their code base. We are no longer compatible with `Aws4Signer`